include/sway/input/text_input_popup.h

```diff
@@ -14,7 +14,8 @@ struct sway_input_popup {
 	struct wl_list link;
 
 	struct wl_listener popup_destroy;
-	struct wl_listener popup_surface_commit;
```

Comment:
Don't we need to still re-constrain the popup when its size changes, for instance?